requirements-dev.txt

```diff
@@ -1,12 +0,0 @@
-flake8==3.3.0
```

Comment:
there is ongoing development for adding support to use `setup.py` file as source of requirements https://github.com/pyupio/pyup/issues/137. Anyway I get the case and will revert that changes